concepts/05 UI Components/DataGrid/70 Custom Keyboard Navigation/05 Change the Default Focused Cell.md

```diff
@@ -0,0 +1,90 @@
+To specify which cell should be focused next when a user navigates through DataGrid, use the [onFocusedCellChanging](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/#onFocusedCellChanging) function. The code below uses `newColumnIndex` to set the column index of the next focused cell.
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $("#dataGridContainer").dxDataGrid({
+        // ...
+        onFocusedCellChanging: function (e) {
+            if (e.columns[e.newColumnIndex].dataField === "myField") { // Checks if the next focused cell will be in the 'myField' column
+                e.newColumnIndex++; // If so, proceed to the next column
```

Comment:
> If so,

We already declared the checked condition in the previous line, didn't we?

```suggestion
                e.newColumnIndex++; // Navigates to the next column
```

Please also keep consistency in mind. In the previous line you said what it does -> **Checks** if ...

Therefore, we should address all other code lines in the same manner.